testdata/populate_metadata_1_tracer_1_struct_from_scratch.bpf.c

```diff
@@ -13,15 +13,12 @@ struct event {
 	__u8 filename[NAME_MAX];
 };
 
-// we need this to make sure the compiler doesn't remove our struct
-const struct event *unusedevent __attribute__((unused));
-
 struct {
 	__uint(type, BPF_MAP_TYPE_PERF_EVENT_ARRAY);
 	__uint(key_size, sizeof(u32));
-	__type(value, struct event);
+	__uint(value_size, sizeof(__u32));
```

Comment:
Sometimes it is `__u32` and some others `u32`. Is there a specific reason for this?

Reply:
There is not any reason. We're already using a mix of them in other places and I copy & pasted the one with __u32. Changed it to be u32 to keep it uniform.

Reply:
In this context, it is fine to use either. But in networking programs, we don't include `vmlinux.h` so `u32` is not defined and we can only use `__u32`.